ftw/simplelayout/browser/client/web/js/app/simplelayout/Block.js

```diff
@@ -49,6 +49,13 @@ define([
       return this;
     };
 
+    this.markLinks = function() {
+      var url = window.location.protocol + "//" + window.location.host;
+      // all http links (without the link-plain class), not within this site
+      $("a[href^=\"http\"]:not(.link-plain):not([href^=\"" + url + "\"])", this.element).attr("target", "_blank");
```

Comment:
Yes. This is `ftw.simplelayout` after all. 😉  The links outside the simplelayout containers are already handled by `mark_special_links.js` (also configured by `external_links_open_new_window`).
This PR fixes the marking for simplelayout containers because they are added/edited dynamically.